pkgs/applications/misc/procmail/default.nix

```diff
@@ -1,22 +1,22 @@
-{ lib, stdenv, fetchurl }:
+{ lib, stdenv, fetchurl, fetchFromGitHub, writeText, bash,
+  msmtp }:
 
 stdenv.mkDerivation rec {
   pname = "procmail";
-  version = "3.22";
+  version = "3.24";
 
-  src = fetchurl {
-    url = "ftp://ftp.fu-berlin.de/pub/unix/mail/procmail/procmail-${version}.tar.gz";
-    sha256 = "05z1c803n5cppkcq99vkyd5myff904lf9sdgynfqngfk9nrpaz08";
+  sendmail = msmtp;
+
```

Comment:
```suggestion
```

Reply:
This was done intentionnally to indicate that any other package delivering the sendmail capability can be used in place of msmtp. If there is a better way to let package users there is flexibility on that package dependency please let me know how to best do that. Alternatively I can add a # comment on the `buildInputs = [ msmtp ]` line.